package.json

```diff
@@ -49,5 +51,11 @@
     "Nate Rajlich <nathan@tootallnate.net> (https://github.com/tootallnate)",
     "Sean McArthur <sean.monstar@gmail.com> (https://github.com/seanmonstar)",
     "Fanie Oosthuysen <fanie.oosthuysen@gmail.com> (https://github.com/weareu)"
-  ]
+  ],
+  "binary": {
+    "module_name": "bcrypt_lib",
+    "module_path": "./lib/binding/",
+    "host": "https://github.com",
+    "remote_path": "/ksmyth/node.bcrypt.js/releases/download/v{version}/"
```

Comment:
do you know if there is a way to reference appveyor directly? Seems a bit of a chicken/egg problem. If I want to make binaries for some version `v`, appveyor will only generate the binary for that after I make the commit bumping the version but then I will need to update the package.json file with the new url to that version?

I suppose one option is to push to github, wait for the appveyor build, download the binary and upload to github releases, and then publish to npm. Is this the workflow you currently use?